caveclient/skeletonservice.py

```diff
@@ -245,6 +245,7 @@ def _build_get_skeleton_endpoint(
         datastack_name: str,
         skeleton_version: int,
         output_format: str,
+        async_: bool,
```

Comment:
I assume this keyword argument is only available after a certain skeletonservice version, so you need to check that the service running is above that version using the decorator infrastructure.